packages/maps/package.json

```diff
@@ -26,7 +26,8 @@
     "emotion-theming": "^9.0.0",
     "prop-types": "^15.6.0",
     "react-emotion": "^9.0.0",
-    "react-google-maps": "^9.4.5"
+    "react-google-maps": "^9.4.5",
+    "rheostat": "^3.0.1"
```

Comment:
web is using 2.0+ & maps is using 3.0+ which may have some breaking changes.
Can we update it later?

Reply:
Since, reactivemap requires reactivesearch to run, this can cause multiple copies issues (not sure)? It'd be best if we update it. We can do it in a separate PR before the `alpha.2` launch (in this week). Please file an issue to monitor the progress on this.